lib/vmpooler/api.rb

```diff
@@ -168,6 +168,45 @@ def hostname_shorten hostname
           JSON.pretty_generate(result)
         end
 
+        get '/status/?' do
+          content_type :json
+
+          result = {}
+          clone_times = []
+
+          result['capacity_current'] = 0
+          result['capacity_total'] = 0
+
+          result['status'] = 1
+
+          $config[:pools].each do |pool|
+            result['capacity_current'] += $redis.scard( 'vmpooler__ready__' + pool['name'] ).to_i
```

Comment:
If this redis key is missing, what happens? Does `capacity_current` retain the original `0` value or does this just throw an error and return a `5xx` status?